certified-connectors/dvelop/apiDefinition.swagger.json

```diff
@@ -835,6 +835,474 @@
+              "itemsPath": "data",
+              "itemTitlePath": "displayName",
+              "parameters": {
+                "defaultAction": {
+                  "value": "createdossier"
+                },
+                "paramDef": {
+                  "value": "k1:queryType|k2:repositoryId"
+                },
+                "k1": {
+                  "value": "docTypeDossier"
+                },
+                "k2": {
+                  "parameterReference": "repositoryId"
+                }
+              }
+            }
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "DefaultGetRequestSchema",
+                "value-path": "request",
+                "parameters": {
+                  "defaultAction": "createdossier",
+                  "paramDef": "k1:repositoryId|k2:docTypeId",
+                  "k1": {
+                    "parameter": "repositoryId"
+                  },
+                  "k2": {
+                    "parameter": "categoryId"
+                  },
+                  "k3": "dummy",
+                  "k4": "dummy",
+                  "k5": "dummy"
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "DefaultGetRequestSchema",
+                "parameters": {
+                  "defaultAction": {
+                    "value": "createdossier"
+                  },
+                  "paramDef": {
+                    "value": "k1:repositoryId|k2:docTypeId"
+                  },
+                  "k1": {
+                    "parameterReference": "repositoryId"
+                  },
+                  "k2": {
+                    "parameterReference": "categoryId"
+                  },
+                  "k3": {
+                    "value": "dummy"
+                  },
+                  "k4": {
+                    "value": "dummy"
+                  },
+                  "k5": {
+                    "value": "dummy"
+                  }
+                },
+                "itemValuePath": "request"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response in case of a successful dossier creation.",
+            "schema": {
+              "x-ms-dynamic-schema": {
+                "operationId": "DefaultGetResponseSchema",
+                "parameters": {
+                  "defaultAction": "createdossier",
+                  "k1": "dummy",
+                  "k2": "dummy",
+                  "k3": "dummy",
+                  "k4": "dummy",
+                  "k5": "dummy"
+                },
+                "value-path": "response"
+              }
+            }
+          },
+          "403": {
+            "$ref": "#/responses/Unauthorized"
```

Comment:
@BeritHorst please add descriptions for your response codes, you can refer to [this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) for an example.